testsuite/test_student_workflow.py

```diff
@@ -59,14 +76,26 @@ def test_submission(client, live_server):
     assert submission.score == 100
     assert submission.total_score == 100
 
+    review_message = '+10.0: Good Job\n-5.0: Bad style\n+0.5:Good Readme'
     client.post(
         '/submission/1/review',
-        data={'review-code': '+10.0: Good Job'},
+        data={'review-code': review_message},
         HTTP_REFERER='/',
     )
 
     submission.refresh_from_db()
 
     assert len(submission.review_message) > 0
-    assert submission.review_score == 10
-    assert submission.total_score == 110
+    assert submission.review_score == 5.5
```

Comment:
Q: Shouldn't the review score be +10? (10 - 5 + 5)

Reply:
The last term is +0.5 not +5.